Source/Mosa.Compiler.Framework/CompilerStages/MapFileStage.cs

```diff
@@ -18,17 +18,17 @@ protected override void Initialization()
 
 	protected override void Finalization()
 	{
-		if (string.IsNullOrEmpty(CompilerSettings.MapFile))
+		if (string.IsNullOrEmpty(MosaSettings.MapFile))
 			return;
 
-		using (var writer = new StreamWriter(CompilerSettings.MapFile))
+		using (var writer = new StreamWriter(MosaSettings.MapFile))
 		{
 			// Emit map file header
-			writer.WriteLine(CompilerSettings.OutputFile);
+			writer.WriteLine(MosaSettings.OutputFile);
 			writer.WriteLine();
-			writer.WriteLine("Timestamp is {0}", DateTime.Now);
+			writer.WriteLine($"Timestamp is {DateTime.Now}");
 			writer.WriteLine();
-			writer.WriteLine("Preferred load address is {0:x16}", Linker.LinkerSettings.BaseAddress);
+			writer.WriteLine($"Preferred load address is {Linker.MosaSettings.BaseAddress:x16}");
```

Comment:
I think using the ``X16`` format (so capitalized) would be more consistent.

Reply:
Leaving as it.